CONTRIBUTING.md

```diff
@@ -1,51 +1,99 @@
-Want to contribute? Great! First, read this page
-(including the "small print" section).
-
 ### Before you contribute
 
-Before we can use your code, you must sign the
-[Google Individual Contributor License Agreement](
-https://cla.developers.google.com/about/google-individual)
-(CLA), which you can do online. The CLA is necessary mainly because you own the
-copyright to your changes, even after your contribution becomes part of our
-codebase, so we need your permission to use and distribute your code. We also
-need to be sure of various other things—for instance that you'll tell us if you
-know that your code infringes on other people's patents. You don't have to sign
-the CLA until after you've submitted your code for review and a member has
-approved it, but you must do it before we can put your code into our codebase.
-Before you start working on a larger contribution, you should get in touch with
-us first through the issue tracker with your idea so that we can help out and
-possibly guide you. Coordinating up front makes it much easier to avoid
-frustration later on.
-
-### Code reviews
-
-All submissions, including submissions by project members, require review. We
-use Github Pull Requests (PRs) for this purpose. We recommend you read [this guide](
-https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
-if you haven't created a Pull Request before.
+Before we can use your code, please fill out either the individual or corporate Contributor License Agreement (CLA).
```

Comment:
Let's mention that the CLA will pop up when you create a PR. You don't need to worry about it before you get started. In fact, let's move the CLA section to *after* the "open a PR" section.